apps/builddao/widget/components/modals/CreateProposal.jsx

```diff
@@ -1,9 +1,11 @@
-const { daos } = VM.require("${config_account}/widget/fetch.daos") || {
-  daos: [],
-};
-
+const DaoSDK = VM.require("sdks.near/widget/SDKs.Sputnik.DaoSDK") || (() => {});
```

Comment:
How about placing this as `"daosdk": "sdks.near/widget/SDKs.Sputnik.DaoSDK"` in aliases (mainnet and testnet) and use `${alias_daosdk}` instead?
```suggestion
const DaoSDK = VM.require("${alias_daosdk") || (() => {});
```